SignalServiceKit/src/Messages/OWSFailedAttachmentDownloadsJob.m

```diff
@@ -121,21 +121,21 @@ - (YapDatabaseSecondaryIndex *)indexDatabaseExtension
 // Useful for tests, don't use in app startup path because it's slow.
 - (void)blockingRegisterDatabaseExtensions
 {
-    [self.storageManager registerExtension:[self indexDatabaseExtension]
+    [self.storageManager registerExtension:[self.class indexDatabaseExtension]
                                   withName:OWSFailedAttachmentDownloadsJobAttachmentStateIndex];
 }
 
-- (void)asyncRegisterDatabaseExtensions
++ (void)asyncRegisterDatabaseExtensionsWithStorageManager:(TSStorageManager *)storageManager
```

Comment:
We were needlessly instantiating some of our "jobs" when we registered their views.